Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/JoinSessionMessage.java

```diff
@@ -26,6 +33,12 @@ public JoinSessionMessage(Status status) {
         settings = null;
     }
 
+    /**
+     * Intializes the JoinSessionMessage class.
+     * 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)